src/Microsoft.DotNet.Analyzers.Compatibility.Tests/Microsoft.DotNet.Analyzers.Compatibility.Tests.csproj

```diff
@@ -24,8 +24,19 @@
     <ProjectReference Include="..\Microsoft.DotNet.Analyzers.Compatibility\Microsoft.DotNet.Analyzers.Compatibility.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="xunit" Version="2.3.1" />
```

Comment:
These are already added automatically by the targets.  I assume you just need `xunit.runner.json` to be copied to the output directory?

Reply:
VS was not finding any tests without the packages references, see below:
```
[5/15/2018 07:31:38 Informational] ------ Run test started ------
[5/15/2018 07:31:41 Warning] [xUnit.net 00:00:00.4605294] Skipping: Microsoft.DotNet.Analyzers.Compatibility.Tests (could not find dependent assembly 'Microsoft.DotNet.Analyzers.Compatibility.Tests, Version=42.42.42')
[5/15/2018 07:31:41 Warning] No test is available in C:\s\github\pjanotti\platform-compat\artifacts\Debug\bin\Microsoft.DotNet.Analyzers.Compatibility.Tests\net461\Microsoft.DotNet.Analyzers.Compatibility.Tests.dll. Make sure that test discoverer & executors are registered and platform & framework version settings are appropriate and try again.
[5/15/2018 07:31:42 Informational] ========== Run test finished: 0 run (0:00:03.3922764) ==========
```